alf/environments/process_environment.py

```diff
@@ -246,7 +246,7 @@ def _worker(self, conn, env_constructor, env_id=None, flatten=False):
         """
         try:
             alf.set_default_device("cpu")
-            env = env_constructor(env_id)
+            env = env_constructor(env_id=env_id)
```

Comment:
This is for a more informative error message when a user writes a load function but forgets to set "env_id" as its argument.